### DISCUSSION TITLE
Little bug I found on the website.

### DISCUSSION THREAD

**Discussion by DanDWD** [availproject/avail#558] [General]
Hello! 

That's not much but there is a little bug/typo in your css file.
here; https://claim.availproject.org/_next/static/css/eaaf786ef00b7991.css
Line 1188, "width" value must be changed for 100% (and not 100vw).

That's all! Have a good day!

**Comment by Leouarz** [comment]
100vw is a valid unit and not a typo

**Reply by DanDWD** [reply]
100 vw displays a scroll bar at the bottom of the page and shifts everything if you drag to the right. 100% fix it.

**Reply by Leouarz** [reply]
Yes, it was done by a colleague and then brought back for some internal reason. Thanks for mentionning thow
